source/Cosmos.System2/Graphics/Bitmap.cs

```diff
@@ -380,6 +380,23 @@ private void CreateBitmap(Stream stream, ColorOrder colorOrder)
             #endregion Pixel Table
         }
 
+        /// <summary>
```

Comment:
I think this message should be moved to the rest of the new methods in `Image.cs`